x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/jira.ts

```diff
@@ -73,12 +75,19 @@ export default function jiraTest({ getService }: FtrProviderContext) {
   };
 
   let jiraSimulatorURL: string = '<could not determine kibana url>';
+  let proxyServer: any;
+  let proxyHaveBeenCalled = false;
 
   describe('Jira', () => {
     before(() => {
       jiraSimulatorURL = kibanaServer.resolveUrl(
         getExternalServiceSimulatorPath(ExternalServiceSimulator.JIRA)
       );
+      proxyServer = getHttpProxyServer(jiraSimulatorURL, () => {
```

Comment:
oh wow, this is pretty cool - I hadn't thought of running the proxy in the FTR!  I was thinking we'd do it in FTS, but this is obviously easier - at least to just run when needed.

Just using it for security+spaces seems good - we'll stress test it more than with spaces_only, and we should run one with the proxy and one without.